guides/common/modules/proc_resolving-package-dependency-errors.adoc

```diff
@@ -29,7 +29,7 @@ If you have successfully installed the {Project} packages, skip this procedure.
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install} _package_name_
+# {client-package-install-el8} _package_name_
```

Comment:
Use `project-*`?

Reply:
Do change because Satellite docs actually use "dnf" here: https://docs.theforeman.org/nightly/Installing_Server_Disconnected/index-satellite.html#resolving-package-dependency-errors_satellite

Reply:
But `project-package-install` would resolve to `dnf install` for Satellite, wouldn't it? The same below.

Reply:
No, this would resolve to "satellite-maintain packages install": https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L87

Reply:
Oh, I see. All good then.